crates/rpc-types-txpool/src/txpool.rs

```diff
@@ -205,7 +204,7 @@ mod tests {
         "type": "0x2",
         "accessList": [],
         "chainId": "0x1",
-        "v": "0x0",
+        "yParity": "0x0",
```

Comment:
breaking change?

Reply:
`v` on non-legacy transactions is deprecated, so that's a serialization change, deser should be fine

though we've decided to test this before merging

Reply:
Related https://github.com/alloy-rs/alloy/pull/865/files/d63e8e1498ab22b038e42ea0456ef9d68ed011ef#r1637954705